io.openems.edge.evcs.ocpp.server/src/io/openems/edge/evcs/ocpp/server/MyJsonServer.java

```diff
@@ -90,6 +90,23 @@ public void newSession(UUID sessionIndex, SessionInformation information) {
 
 				var ocppIdentifier = information.getIdentifier().replace("/", "");
 
+				// Check if there is already a session for this ocppIdentifier
```

Comment:
Over the weekend, we tested both our patch(https://github.com/OpenEMS/openems/pull/2908#issuecomment-2515850362) and this one, and both appeared to work properly. However, I’m concerned about the following point in this patch. What do you think?

- The old session is deleted before establishing a new one(https://github.com/OpenEMS/openems/pull/2908/files#diff-a9d06149a614c84998d6bcad6d51f36ca40834f2e200120f2a2e9bfb290601eaR119). If something goes wrong during that process, wouldn’t we end up unable to use both the old and new sessions?